src/common/navigation.yaml

```diff
@@ -56,8 +57,9 @@ sections:
         sections:
           - title: Tutorials
             pages:
+              - path: /integrate-stacking
               - path: /install-api
-              - path: /installing-memcached
+              - path: /install-memcached
```

Comment:
We have to be very careful with renaming paths. We need to create a redirect anytime we do for a page that has been active.

Reply:
great point, adding a redirect!